python/plugins/processing/algs/gdal/warp.py

```diff
@@ -177,7 +177,7 @@ def getConsoleCommands(self):
             arguments.extend(rastext)
 
         if GdalUtils.version() >= 2000000:
-            if rastext and rastext_crs is not None:
+            if rastext and len(rastext_crs) > 0:
```

Comment:
Would it be more straightforward to cast this to a bool? This would not raise an exception in case it's still `None` in some places

    if rastext and rastext_crs: